pubsub/rabbitmq/metadata.go

```diff
@@ -223,6 +226,15 @@ func createMetadata(pubSubMetadata pubsub.Metadata, log logger.Logger) (*metadat
 		return &result, fmt.Errorf("%s invalid TLS configuration: %w", errorMessagePrefix, err)
 	}
 
+	if val, found := pubSubMetadata.Properties[metadataSaslExternal]; found && val != "" {
+		if boolVal, err := strconv.ParseBool(val); err == nil {
```

Comment:
Instead of `ParseBool` please use our `utils.IsTruthy` function https://github.com/dapr/components-contrib/blob/master/internal/utils/utils.go#L26

Reply:
Thanks, I replaced it. ✌️